### PR TITLE
Use Hot Wallet for stacking

### LINKED ISSUE #134: add delegate-stx-extend / -increase for whales
Users with a cold wallet and a hot wallet should be able to extend and increase their stacking.

- [x] Add path `/pool-admin/delegate-stack-extend` and `/pool-admin/delegate-stack-increase`
- [x] Add path `/pool-admin` to see the status of the hot wallet(s) and buttons for possible actions

### PR DESCRIPTION
**Pull request by friedger**
This PR
* adds extend and increase actions for solo stacking users that use a hot/cold wallet setup.
* adds a link "Use Hot Wallet" at the footer of the main page
* fixes #134 
* prepares for more pool info on a separate path like /pool/ST1234

Quick video of the flow: https://www.loom.com/share/d1983a0ae1e54db6b4f1e63611ac1893?sid=5caffce3-e10b-4634-bd63-a0297432f3ed

### CODE REVIEW

**Comment by markmhx** [comment]
This "Hot Wallet" feature is essentially for pool operators who use the standardized proxy contracts, yea (or does it work for all pools regardless of that usage)?

My sense is that we could name this feature more clearly / explicitly (e.g. "Manage pool"?), as well as add some intro information to the "Manage your pool" page so visitors know what they're looking at (and can determine whether it's relevant to them). E.g. by adding a paragraph or two to the top of that page.

cc @mica000

**Comment by friedger** [comment]
@markmhx 
The use case is for solo stackers/whales that hold their main amount in a cold wallet and don't want to touch often. They delegate once using normal pox contract to their hot wallet.
Then they use their hot wallet on lockstacks.com to lock/extend and stack aggregation commit. They don't use a wrapper because they are their only pool members.

Some Whale FAQ are here https://blog.friedger.de/posts/MeBQ49LDcr_ZD_afEwOwH#whale-faq

File: src/pages/stacking/pool-admin/delegate-stack-increase/delegate-stack-increase.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  if (
+    getSecondsUntilNextCycleQuery.isError ||
+    typeof getSecondsUntilNextCycleQuery.data !== 'number' ||
+    getPoxInfoQuery.isError ||
+    !getPoxInfoQuery.data ||
+    getAccountExtendedBalancesQuery.isError ||
+    typeof getAccountExtendedBalancesQuery.data.stx.balance !== 'string'
+  ) {
+    const msg = 'Failed to load necessary data.';
+    const id = '8c12f6b2-c839-4813-8471-b0fd542b845f';
```
Comment (kyranjamie) [inline src/pages/stacking/pool-admin/delegate-stack-increase/delegate-stack-increase.tsx:85]: hardcoded id?  `window.crypto.randomUUID()`

**Review by kyranjamie — APPROVED** [review]

File: src/pages/stacking/pool-admin/delegate-stack-increase/delegate-stack-increase.tsx (hunk 1 above)
Comment (friedger) [reply src/pages/stacking/pool-admin/delegate-stack-increase/delegate-stack-increase.tsx:85]: There are many of these in the code base. What is the purpose? How are they defined?

**Comment by mica000** [comment]
Not sure I fully understand what is the use case in here but also missing what a hot and cold wallet means

**Comment by friedger** [comment]
A hot wallet is a wallet for your day to day use with small amounts. A cold wallet is for your big savings.

A cold wallet is for example a multisig wallet and it can be more difficult to sign a smart contract transaction because of additional security rules.

If you trust only Hiro Desktop wallet for your cold wallet, you can't sign pox wrapper transactions.

**Comment by friedger** [comment]
@markmhx Some description like this?

image

**Pull request merged**